hiive/models/staging/stg_customer_order_summary.sql

```diff
@@ -0,0 +1,67 @@
+{{
+    config(enabled=false)
+}}
+
+
+WITH raw_customers AS (
+   SELECT
+        id AS customer_id,
+        first_name,
+        last_name,
+        email_address,
+        phone_number,
+        created_at AS signup_date
+   FROM raw.database.orders
```

Comment:
This should be a `ref('')`, not a table name.

Reply:
Also, CTE is named raw_customers, but the table used is `orders`, probably this should be `users`.